homeassistant/components/sensor/device_condition.py

```diff
@@ -78,6 +82,7 @@
 CONF_IS_VALUE = "is_value"
 
 ENTITY_CONDITIONS = {
+    DEVICE_CLASS_APPARENT_POWER: [{CONF_TYPE: CONF_IS_APPARENT_POWER}],
```

Comment:
Should we use the enum here instead?

Reply:
Yeah, we should.
Maybe better to refactor first, then rebase this PR?

Reply:
Until now I have stayed away from the core entity platforms, and I think the other user who has kind of taken over has avoided it also.

Reply:
OK, I'll do that then

Reply:
Refactoring done + this PR rebased.